<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsPage</class>
 <widget class="QWidget" name="StatisticsPage">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digging</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="gridFrame_2">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="horizontalSpacing">
         <number>90</number>
        </property>
        <property name="verticalSpacing">
         <number>18</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Block Height:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="heightBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>00000</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PoW Block Reward:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="rewardBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>250</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PoW Difficulty:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="diffBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0.0002</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PoS Difficulty:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="diffBox2">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0.0002</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Net Hashrate:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="pawrateBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0 MH/s</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label3_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Last Block Time:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="localBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label1_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Connections:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="connectionBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total BERN Volume:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="volumeBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>250</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your Stake Weight:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="minBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Network Stake Weight:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="maxBox">
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>7</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans,sans-serif</family>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18px; font-weight:600; color:#464747;&quot;&gt;BERN Network Statistics&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>450</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;color:#464747;&quot;&gt;Mining Phases&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="phaselbl1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>phaselbl1</string>
      </property>
     </widget>
     <widget class="QLabel" name="phaselbl2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>phaselbl2</string>
      </property>
     </widget>
     <widget class="QLabel" name="phaselbl3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>phaselbl3</string>
      </property>
     </widget>
     <widget class="QLabel" name="phaselbl4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>phaselbl4</string>
      </property>
     </widget>
     <widget class="QLabel" name="phaselbl5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>phaselbl5</string>
      </property>
     </widget>
     <widget class="QLabel" name="phaselbl5_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;* PoS will always be 10% Annual. &lt;br/&gt;24 hour minimum, 4 year maximum stake age.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="phaselbl5_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;b&gt;(Bold&lt;/b&gt; denotes current phase.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="startButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>225</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#333333; color:#cecece;</string>
     </property>
     <property name="text">
      <string>Update Statistics</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
